packages/twenty-server/src/core/auth/emails/password-reset-link.email.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Link } from '@react-email/components';
+
+import { MainText } from 'src/emails/components/MainText';
+import { Title } from 'src/emails/components/Title';
+import { BaseEmail } from 'src/emails/components/BaseEmail';
+import { CallToAction } from 'src/emails/components/CallToAction';
+
+type PasswordResetLinkEmailData = {
+    userName: string;
+    duration: string;
+    link: string;
+};
+
+export const PasswordResetLinkEmail = ({
+    userName,
+    duration,
+    link,
+}: PasswordResetLinkEmailData) => {
+    const helloString = userName?.length > 1 ? `Hello ${userName}` : 'Hello';
+    return (
+        <BaseEmail>
+            <Title value="Reset your password" />
+            <CallToAction href={link} value="Reset" />
+            <MainText>
+                {helloString},
+                <br />
+                This link is only valid for the next {duration}.
+                If link does not work, you can use reset link directly:
+                <br />
+                <Link href={link}>{link}</Link>
```

Comment:
Think we will need a Custom EmailLink component for twenty emails to match Figma design (we should have gray links not blue links) 
![image](https://github.com/twentyhq/twenty/assets/29927851/cb9bc10b-c93b-410c-9ca2-67af3f60a2db)

Reply:
Have created a custom component for the link.